test/with-express-json.test.ts

```diff
@@ -3,50 +3,52 @@
  */
 import fetch from 'node-fetch'
 import express from 'express'
-import { HttpServer } from '@open-draft/test-server/http'
 import { HttpResponse, http } from 'msw'
 import { createMiddleware } from '../src'
 
-const httpServer = new HttpServer((app) => {
-  // Apply a request body JSON middleware.
-  app.use(express.json())
-
-  app.use(
-    createMiddleware(
-      http.post<never, { firstName: string }>('/user', async ({ request }) => {
-        const { firstName } = await request.json()
-
-        return HttpResponse.json(
-          { firstName },
-          {
-            headers: {
-              'x-my-header': 'value',
-            },
+import httpModule from 'http'
+import { AddressInfo } from 'net'
+
+const app = express()
+
+app.use(express.raw({ type: '*/*' }))
```

Comment:
What do you think if we bake in `express.raw()` into `createMiddleware()` itself?

Applying it to the entire server is too much anyway since people can attach handlers-based routes to an existing Express server.